pkg/k8s/client.go

```diff
@@ -161,3 +162,11 @@ func NewAsBackplaneClusterAdmin(clusterID string, options client.Options, elevat
 
 	return client.New(cfg, options)
 }
+
+func GetCurrentCluster() (string, error) {
```

Comment:
unrelated to this specific PR - we could probably incorporate this function into a LOT more commands.